fuzz/fuzz_targets/blake3.rs

```diff
@@ -0,0 +1,23 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use arbitrary::{Arbitrary, Unstructured, Result};
+use bitvm::hash::blake3_u4_compact::test_blake3_compact_givenbyteslice;
```

Comment:
Does this work with `cargo check` for you? Seems like this doesn't play nicely with the target restriction :(